test/statsCases/preset-normal/expected.txt

```diff
@@ -1,4 +1,5 @@
 Hash: c5a6856b43905ae12f17
+Environment (--env): None
```

Comment:
It would be better to omit this line if the `--env` is not used. There is no benefit in adding it.